pkg/query-service/app/clickhouseReader/reader.go

```diff
@@ -1983,6 +1993,9 @@ func (r *ClickHouseReader) SearchTraces(ctx context.Context, params *model.Searc
 		}
 	}
 
+	searchSpansResult[0].StartTimestampMillis = startTime - durationNano
```

Comment:
@srikanthccv Here we should have converted durationNano to milliseconds before subtracting or adding